src/client/components/layout/AppRouter.tsx

```diff
@@ -58,6 +58,7 @@ const AppRouter: FunctionComponent = (): ReactElement => {
             '/course-admin',
             '/faculty-admin',
             '/room-admin',
+            '/room-schedule',
```

Comment:
I'm a little confused about this - isn't it already hidden?

Reply:
So the Router block from 45-67 is handling the routing only when the user loads `info.seas.harvard.edu`, and it is already hidden there.

The second Router block, from 73-113, handles the `planning.seas.harvard.edu` routing. And it's definitely an edge case, but if a user somehow was logged into planning but didn't have read-only permission, we still want to block the Room Schedule (but not the four-year-plan or regular schedule). In theory, this should be prevented by Grouper, so it's kind a belt-and-suspenders check.

Reply:
I see. The docblock for `isLoggedIn` says: _"True if the user is permitted to view protected content within the course-planning app. If this is false and isLoggedIn is true, the user may be missing group memberships. A value of true also implies that isLoggedIn is true."_ so I kind of figured that this was (more or less) impossible.

Reply:
Either way, I've made that change in c9ef0a0889e695a9ea44b9f16b4815be0e9afb39

Reply:
Just to clarify, you pasted the docblock for `isReadOnly`. 

It is _technically_ possible for `isLoggedIn` to be `true` while `isReadOnly` is `false. In practice, we have a Harvard Key Authorization filter in place that only allows users with `Read-Only` permission to authenticate, but that's external to the application. We want to make sure we're also applying internal authorization to prevent external changes from breaking the app.

Reply:
Sorry, I was looking at the wrong docblock. You're right.